README.md

```diff
@@ -54,9 +54,10 @@
 
 # LTS Plan
 
-New semver-major releases of Node.js are cut from `master` every six months.
-New even-numbered versions (e.g. v6, v8, v10, etc) are cut in April. New 
-odd-numbered versions (e.g. v5, v7, v9) are cut in October.
+New semver-major releases of Node.js are cut from `master` every six months
+and become the "current" release.  New even-numbered versions (e.g.  v6, v8,
+v10, etc) are cut in April.  New odd-numbered versions (e.g.  v5, v7, v9)
+are cut in October.
```

Comment:
Suspect any additional detail on the criteria that is required in this particular doc should come have input from @TheAlphaNerd, @Fishrock123 or others. We could add a phrase about it being at the discretion from the release team taking into account the labels on the original PRs